Dockerfile

```diff
@@ -3,6 +3,9 @@ FROM fluent/fluentd:edge
 
 ENV FLUENTD_DIR=fluentd
 
+# Setting default values
```

Comment:
What happens if you run without docker?

The default value should be applied somewhere else, at the moment sounds like a better choice the ruby script itself.